src/towncrier/newsfragments/482.feature.rst

```diff
@@ -0,0 +1 @@
+If no filename is given when doing ``towncrier`` create, interactively ask for the issue number and fragment type (and then launch an interactive editor for the fragment content).
```

Comment:
I think that the `--add-extension` is also a new feaure introduced by this PR.

Also, `--eof-neline` is a new feature of this PR.

Should't we have release notes for these features ?